Source/JavaScriptCore/runtime/Options.cpp

```diff
@@ -747,7 +747,7 @@ void Options::notifyOptionsChanged()
 
     // Do range checks where needed and make corrections to the options:
     ASSERT(Options::thresholdForOptimizeAfterLongWarmUp() >= Options::thresholdForOptimizeAfterWarmUp());
-    ASSERT(Options::thresholdForOptimizeAfterWarmUp() >= Options::thresholdForOptimizeSoon());
+    // ASSERT(Options::thresholdForOptimizeAfterWarmUp() >= Options::thresholdForOptimizeSoon());
```

Comment:
Why comment it out instead of removing it?  Also, in the commit msg, it would be good to explain why this should be removed.